pkgs/development/tools/xcbuild/sdk.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, writeText, toolchainName, sdkName, xcbuild }:
+
+let
+
+  SDKSettings = {
+    CanonicalName = sdkName;
+    DisplayName = sdkName;
+    Toolchains = [ toolchainName ];
+    Version = "10.9";
+    MaximumDeploymentTarget = "10.9";
```

Comment:
I'd aim for 10.10, since that's the version we're using elsewhere. Or if you can pull it in from `MACOSX_DEPLOYMENT_TARGET`, that'd be handy too. Or if not, we could designate a central file in stdenv/darwin/ and perhaps call it target-version.nix, so we can only change it in one place and have it update everywhere.